src/commands/info.json

```diff
@@ -26,6 +26,7 @@
             "RESPONSE_POLICY:SPECIAL"
         ],
         "reply_schema": {
+            "description": "The info is divided into section/sections in the form of #<section name> and below the section name, a map between named fields and values in the form of <field>:<value> lines separated by newlines composed.",
```

Comment:
I think we can also mention the case of multi-field lines (like db0).
Either that, or maybe go the other way and simplify it to just mention its a list of lines with one field per line (avoid the part about sections which is usually just skipped by parsers)